terraform.sample.yml

```diff
@@ -19,6 +19,7 @@
 # `roles/` directory, or by checking the online documentation at
 # microservices-infrastructure.readthedocs.org.
 - hosts: all
+  sudo: yes
```

Comment:
please remove this. We should use sudo on tasks only as they need it, never on all tasks in a playbook.